opensrp-reveal/src/main/java/org/smartregister/reveal/util/RevealJsonFormUtils.java

```diff
@@ -522,7 +524,11 @@ public void populateFormWithServerOptions(String formName, JSONObject formJSON)
                             dataCollector);
                 }
 
-                populateUserAssignedLocations(formJSON, JsonForm.ZONE, Arrays.asList(OPERATIONAL_AREA, ZONE));
+                if (MACEPA_PROVINCES.contains(PreferencesUtil.getInstance().getCurrentProvince())) {
+                    populateUserAssignedLocations(formJSON, JsonForm.ZONE, Arrays.asList(HEALTH_CENTER));
```

Comment:
Shouldn't this be district or the issue is outdated

Reply:
The issue was updated in this comment https://github.com/OpenSRP/opensrp-client-reveal/issues/1139#issuecomment-722405670